service/controller/resource/ipam/create.go

```diff
@@ -4,6 +4,9 @@ import (
 	"context"
 	"fmt"
 	"net"
+	"strconv"
+
+	awsoperatorannotation "github.com/giantswarm/aws-operator/pkg/annotation"
```

Comment:
Can you put the import to the other project imports below? Also why not just `annotation` without the alias?

Reply:
yeah, I got confused as initially that imported a different annotation package but it's fine now